extra/utils/permissions.js

```diff
@@ -0,0 +1,32 @@
+const permissions = {
+  getUsers1: {
+    all: ["head-trainer"],
+    read: ["trainee", "trainer"],
+    write: ["trainer"],
+    delete: []
+  },
+  getUsers2: {
+    all: ["head-trainer"],
+    read: ["trainee", "trainer"],
+    write: ["trainer"],
+    delete: []
+  }
+};
+
+function hasPermission(moduleName, role, permissionType) {
```

Comment:
If module, type and role is provided other than the mentioned one then there is no case to handle that type of scenario.